integreat_cms/cms/templates/_collapsible_box.html

```diff
@@ -15,7 +15,7 @@
            class="down-arrow {% if not collapsed %}hidden{% endif %}"></i>
     </div>
     <div class="collapsible-content transition-all duration-200 ease-in-out overflow-hidden {% if collapsed %}h-0{% endif %}">
-        <div class="p-4">
+        <div class="{% if not no_container %} p-4 {% endif %}">
```

Comment:
```suggestion
        <div class="{% if not no_padding %} p-4 {% endif %}">
```
I find it a bit confusing that the `container`/`no_container` variables are not mutually exclusive and are used for two different things, meaning that at the moment a box can be neither of them or both, which makes no sense from a yes/no perspective.